RevenueCatUI/CustomerCenter/Data/CustomerCenterActionWrapper.swift

```diff
@@ -151,3 +153,85 @@ extension CustomerCenterConfigData.HelpPath {
     }
 
 }
+
+#if os(iOS)
+
+/// Internal Combine-based wiring for handling `CustomerCenterInternalAction` events.
+/// These methods attach handlers to internal publishers and are used internally
+/// by any ViewModel to bridge actions to business logic.
+///
+/// - Important: This API is intended **for internal use only** within the SDK.
```

Comment:
Why this scare message? 😆

Reply:
I am thinking on how to 100% delete the preference use. The tricky part is that for the public API, we rely on preference keys so I am not sure how to avoid it yet.

One idea I have is using environment functions instead of preference keys, but that's not great either.

Reply:
Oh, right. I see what you mean... 🤔 
I really wouldn't know how to easily solve that, tbh. I do like the idea of environment functions **but** I'm fear/wonder that it could have the same limitations as `preference` keys...? I don't really know, but yeah, we should perhaps try that approach